src/components/ChartView/ChartVersionsList.tsx

```diff
@@ -1,22 +1,47 @@
 import * as React from "react";
+import { Link } from "react-router-dom";
 
 import { IChartVersion } from "../../shared/types";
+import * as url from "../../shared/url";
 
 interface IChartVersionsListProps {
+  selected: IChartVersion;
   versions: IChartVersion[];
 }
 
-class ChartVersionsList extends React.Component<IChartVersionsListProps> {
+interface IChartVersionsListState {
+  showAll: boolean;
+}
+
+class ChartVersionsList extends React.Component<IChartVersionsListProps, IChartVersionsListState> {
+  public state: IChartVersionsListState = {
+    showAll: false,
+  };
+
   public render() {
-    const items = this.props.versions.slice(0, 5).map(v => (
-      <li key={v.id}>
-        {v.attributes.version} - {this.formatDate(v.attributes.created)}
-      </li>
-    ));
+    const versions = this.state.showAll ? this.props.versions : this.props.versions.slice(0, 5);
+    const items = versions.map(v => {
+      const selectedClass =
+        this.props.selected.attributes.version === v.attributes.version
+          ? "type-bold type-color-action"
+          : "";
+      return (
+        <li key={v.id}>
+          <Link className={selectedClass} to={url.app.charts.version(v)}>
+            {v.attributes.version} - {this.formatDate(v.attributes.created)}
+          </Link>
+        </li>
+      );
+    });
     return (
       <div className="ChartVersionsList">
         <ul className="remove-style padding-l-reset margin-b-reset">{items}</ul>
-        <span className="type-small">Show all...</span>
+        {!this.state.showAll &&
+          this.props.versions.length > 5 && (
+            <span className="type-small" onClick={this.handleShowAll}>
+              Show all...
```

Comment:
"show all" is a clickable link, we should style it appropriately.

Reply:
👍  done